test/CMakeLists.txt

```diff
@@ -3,22 +3,24 @@
 ###############
 
 LIST(APPEND axl_test_srcs
-    test_axl.c
-    test_axl_sync.c
-    test_axl_async_daemon.c
-    test_axl_async_ibmbb.c
+    axl_cp.c
 )
 
-ADD_EXECUTABLE(test_axl ${axl_test_srcs})
-TARGET_LINK_LIBRARIES(test_axl axl)
+ADD_EXECUTABLE(axl_cp ${axl_test_srcs})
+
+find_package(Threads REQUIRED)
```

Comment:
The tests shouldn't need to find pthreads unless the test code itself is using the pthread library.

Reply:
Good catch.  I'll fix it in my next push

Reply:
Same with the `TARGET_LINK_LIBRARIES`, the test should get the link to pthread through linking to AXL. If that isn't working we have to change the cmake in src.